src/my_app/lib/counter/view/counter_rotary_listener.dart

```diff
@@ -0,0 +1,52 @@
+import 'dart:async';
+
+import 'package:flutter/material.dart';
+import 'package:flutter_bloc/flutter_bloc.dart';
+import 'package:very_good_wearos_app/counter/cubit/counter_cubit.dart';
+import 'package:wearable_rotary/wearable_rotary.dart' as wearable_rotary;
+import 'package:wearable_rotary/wearable_rotary.dart' hide rotaryEvents;
+
+/// A [StatefulWidget] that parses events on [rotaryEvents] and updates
+/// the [CounterCubit].
+class CounterRotaryListener extends StatefulWidget {
```

Comment:
This works fine but it seems to me that the `wearable_rotatory` should have a widget similar to a `GestureDetector` where one passes an `onTap`.

I would expect an API as such:
```dart
RotatoryDetector(
  onRotate: (details) => {}
);
```
I opened an issue regarding [this](https://github.com/flutter-tizen/plugins/issues/544).

However, maybe we can use the [`RotatoryScrollController`](https://pub.dev/documentation/wearable_rotary/latest/wearable_rotary/RotaryScrollController-class.html) and make the `CounterView` stateful? [This](https://github.com/flutter-tizen/plugins/blob/d7cac5ce61725fcda5f9ee8c8eecd2b7a6caf341/packages/wearable_rotary/lib/src/rotary_scroll_controller.dart#L20) code might be useful.

Reply:
Well, I ended up merging this widget to CounterView, seems like the scroll view thing could be clearer and more manageable since it is just a screen we have to listen to.